tests/nativeapp/test_manager.py

```diff
@@ -11,7 +16,7 @@
     ensure_correct_owner,
 )
 from snowflake.cli.plugins.object.stage.diff import DiffResult
-from snowflake.cli.api.project.definition_manager import DefinitionManager
+from snowflake.connector import ProgrammingError
```

Comment:
Curious how the test was working without importing `ProgrammingError` ? Was `from tests.testing_utils.fixtures import *` importing it?